app/models/pb_core.rb

```diff
@@ -28,6 +28,16 @@ def asset_title
     @asset_title ||= xpath_optional('/*/pbcoreTitle[@titleType="Open Vault Title"]')
   end
 
+  def title
+    @title ||= [series_title, program_title, asset_title].select { |x| x }.join('; ')
```

Comment:
maybe `.select { |x| !x.nil? }` ... or..  `.reject(&:nil?)` would be more explicit... but oh well. It's merged.